### PR TITLE
Mobx search breadcrumb

### LINKED ISSUE #4169: Data search result breadcrumbs
As a map user, I want to see the hierarchy of where my data catalogue search results sit so I can find it again in the tree

**Acceptance criteria:**
- [x] Clicking a search result of a catalog item shows the "breadcrumbs" of the selected catalog item at the bottom of the data catalogue
- [x] Breadcrumb has a globe icon
- [x] Breadcrumb background uses `$grey` from theme
- [x] Breadcrumb text uses `$dark` from theme
- [x] Clicking any of the parents of those breadcrumbs brings you out of search results, into the catalog with that group selected

(? To clarify with Vic) what does it mean to have a search result of a group like "Transport" or "NSW Digital Twin Secured"

### PR DESCRIPTION
**Pull request by reginapramesti**
### What this PR does

Fixes #4169 

This PR adds the breadcrumbs showing the hierarchy to the selected catalogue when searching through the catalogue.

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by soyarsauce** [comment]
IE11 strikes again, in the form of a saucelabs failure - restarting build
Screen Shot 2020-06-11 at 11 08 57 am

File: lib/Language/en/translation.json [hunk 1]
```diff
@@ -190,7 +190,7 @@
   },
   "helpPanel": {
     "menuPaneTitle": "We're here to help",
-    "menuPaneBody": "Find useful tips on how to use TerriaJS by either checking the guides below or by contacting the team at [info@terria.io](mailto:info@terria.io)"
+    "menuPaneBody": "Find useful tips on how to use the Digital Twin by either checking the video guides below or by contacting the team at [info@terria.io](mailto:info@terria.io)"
```
Comment (soyarsauce) [inline lib/Language/en/translation.json]: Just writing here to revert this one incase someone misses during review when it's ready

Comment (reginapramesti) [reply lib/Language/en/translation.json]: Ah whoops! Must've carried over from the old days when I updated the branch

**Comment by reginapramesti** [comment]
Note I used `$greyLighter` for the background because it gives a better contrast than a `$grey` background, and I think it resembles the designs a bit better

**Comment by soyarsauce** [comment]
@reginapramesti this is so awesome!! But is not quite there yet, maybe spend a small amount of time to figure out why a subgroup doesn't show the full breadcrumb until you get deep into a catalog item.

If you can't figure it out quickly, I would have liked to merge this in its current state and raise another ticket for this problem specifically - however it changes `getPath` and I don't want it to break that - _unless_ mobx behaviour already currently does the below without the changes in this PR.

(e.g. in the first image, it should show `Live Data -> NSW Live Transport` but it only shows `NSW Live Transport`)

image


image

**Comment by soyarsauce** [comment]
Tests are broken here but let's park this until after deliverables

**Review by soyarsauce — APPROVED** [review]

**Pull request merged**